### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by hauteballe**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+let numbers = document.querySelectorAll('.btn-nmb'),
```
Comment (tibmt) [inline script.js:1]: Change `let` to 'const` for unchangeable variables.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    operations = document.querySelectorAll('.btn-operation'),
```
Comment (tibmt) [inline script.js:2]: Add separate `const` and `let` keyword for every variable.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+};
+
+const decimal = () => {
+    let localDecimalMemory = display.value;
+    
+    if (memoryNewNumber) {
+        localDecimalMemory = '0.';
+        memoryNewNumber = false;
+    } else {
+        if (localDecimalMemory.indexOf('.') === -1) {
```
Comment (tibmt) [inline script.js:55]: Please, use this

https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/includes

File: script.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    let clearButton = clearButtons[i];
+    clearButton.addEventListener("click", function(e) {
+    clear(e.target.textContent);
+    });
+};
+
+decimalButton.addEventListener("click", decimal);
+
+result.addEventListener("click", function(e){
+    console.log("Клик по result")
```
Comment (tibmt) [inline script.js:98]: Use only english language

File: script.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        memoryNewNumber = true;
+    } else if (id === 'c') {
+        display.value = '0'; 
+        memoryNewNumber = true;
+        memoryCurrentNumber = 0;
+        memoryPendingOperation = '';
+    }
+};
+
+for (let i = 0; i<numbers.length; i++) {
```
Comment (tibmt) [inline script.js:74]: Implement event delegation, please

Like here https://learn.javascript.ru/event-delegation

File: script.js (hunk 1 above)
Comment (hauteballe) [reply script.js:1]: Done

File: script.js (hunk 2 above)
Comment (hauteballe) [reply script.js:2]: Done

File: script.js (hunk 3 above)
Comment (hauteballe) [reply script.js:55]: Done

**Pull request merged**